components/dot_menu/dot_menu.tsx

```diff
@@ -464,9 +464,7 @@ export class DotMenuClass extends React.PureComponent<Props, State> {
             </span>
         );
 
-        const fromWebhook = this.props.post.props?.from_webhook === 'true';
```

Comment:
we have `isFromWebhook` "utils/post_utils.ts". May be we can even add `isFromBot` check in that utils too.

Reply:
@M-ZubairAhmed done and I updated some use cases.